README.md

```diff
@@ -35,6 +35,16 @@ Run `target-postgres` against a [Singer](https://singer.io) stream.
 
 It ignores "STATE" type Singer messages.
 
+### Configuration
```

Comment:
Can we either remove this header and bump "Invalid records" up a level, or create a true configuration section with a table of all configuration file key/value options?

Reply:
Sounds good. I'd personally lean towards the table. Should we use JSON Schema syntax for the types etc.?